rl4co/models/zoo/matnet/decoder.py

```diff
@@ -21,8 +21,8 @@ class PrecomputedCache:
     logit_key: Tensor
 
 
-class MatNetDecoder(AutoregressiveDecoder):
-    def _precompute_cache(self, embeddings: Tuple[Tensor, Tensor], td: TensorDict = None):
+class MatNetDecoder(AttentionModelDecoder):
+    def _precompute_cache(self, embeddings: Tuple[Tensor, Tensor], *args, **kwargs):
```

Comment:
It seems to me `*args` and `**kwargs` are never actually used or handed down - why keep them, then?

Reply:
We keep `**kwargs` because we have some default values in the `AutoregressivePolicy`